easybuild/easyconfigs/n/netCDF/netCDF-4.6.0-intel-2018a-HDF5-1.10.1.eb

```diff
@@ -0,0 +1,36 @@
+name = 'netCDF'
+version = '4.6.0'
+hdf5_ver = '1.10.1'
+versionsuffix = '-HDF5-%s' % hdf5_ver
```

Comment:
this one can be removed too, just use the existing `netCDF-4.6.0-intel-2018a.eb`